src/shared/localizedText.ts

```diff
@@ -13,6 +13,7 @@ export const ok = localize('AWS.generic.response.ok', 'OK')
 export const retry = localize('AWS.generic.response.retry', 'Retry')
 export const skip = localize('AWS.generic.response.skip', 'Skip')
 export const confirm = localize('AWS.generic.confirm', 'Confirm')
+export const proceed = localize('AWS.generic.proceed', 'Proceed')
```

Comment:
"Continue" is a more common verb, no?

Reply:
"Proceed" was the specific word Ankit used in the ticket. I'm assuming it was intentional

Reply:
Now updated to "Continue"